packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -206,6 +207,8 @@ const Autocomplete = React.memo(props => {
     const onSelectItem = item => {
         if (!item) return;
 
+        if (item?.disabled && props.should_ignore_on_select) return;
```

Comment:
Since we already breakdown `should_ignore_on_select` on line 39, we don't need to access props object here

Reply:
```suggestion
        if (item?.disabled && should_ignore_on_select) return;
```

Reply:
done can you please approve?:)